cmd/common-main.go

```diff
@@ -48,6 +48,7 @@ import (
 	"github.com/inconshreveable/mousetrap"
 	dns2 "github.com/miekg/dns"
 	"github.com/minio/cli"
+	consoleoauth2 "github.com/minio/console/pkg/auth/idp/oauth2"
```

Comment:
why is this `consoleoauth2` ?

Reply:
Just to indicate that it's from console's sub-package.